packages/dynamite/dynamite_end_to_end_test/lib/types.openapi.dart

```diff
@@ -70,6 +70,56 @@ abstract class Base implements $BaseInterface, Built<Base, BaseBuilder> {
   static Serializer<Base> get serializer => _$baseSerializer;
 }
 
+@BuiltValue(instantiable: false)
+abstract interface class $AdditionalPropertiesInterface {
+  @BuiltValueField(wireName: 'empty_schema_bool')
+  BuiltMap<String, JsonObject>? get emptySchemaBool;
+  @BuiltValueField(wireName: 'empty_schema')
+  BuiltMap<String, JsonObject>? get emptySchema;
+  BuiltMap<String, BuiltMap<String, JsonObject>>? get nested;
```

Comment:
I think that nested is the only broken one rn.
And iirc the nextcloud specs make use of that pattern

Reply:
Yep there are some places this appears. Why is it broken though, this code looks fine to me?

Reply:
What about this?

Reply:
You where right :)
I could not find anything that indicated this is actually wrong.